website/src/docs/hotchocolate/filtering/filter-and-operations.md

```diff
@@ -0,0 +1,47 @@
+# Filter & Operations Kinds
```

Comment:
Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.